cms/djangoapps/import_from_modulestore/tests/test_api.py

```diff
@@ -1,102 +1,109 @@
 """
```

Comment:
Why are we changing functional tests to unit tests again, does it make sense?

Reply:
It seems to me that it's better to have a couple of functional tests that will check if everything works after changes than just protect the code from changes, as unit tests do now.